cmd/importer/db.go

```diff
@@ -209,6 +210,32 @@ func genColumnData(table *table, column *column) (string, error) {
 
 		data = append(data, '\'')
 		return string(data), nil
+	case mysql.TypeNewDecimal:
+		var data string
+		var limit = int64(math.Pow10(tp.Flen))
```

Comment:
When `math.Pow10` is larger than max int64, cast it to int64 will get a negative value.